tasks/main.yml

```diff
@@ -25,6 +25,14 @@
     src: burp.conf.j2
     dest: "{{ burp_client_dir }}/burp.conf"
 
+- name: place the zfs snapshot-backup-over-a-fifo script on the client
+  copy:
+    dest: "{{ etc_dir }}/burp/zfs_script"
+    src: "zfs_script"
+    owner: root
+    group: "{{ root_group }}"
+    mode: 0755
```

Comment:
when statment missing, only copy when burp_client_backup_script is true, using | bool.

Reply:
correct!